### DISCUSSION TITLE
課題3についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#52] [Q&A]
課題13の「i」と[j]をそれぞれコンソールに出力してみたのですが、
「i」は、１～１９の数字が大きいもの順で並んでいて、
「j」は、０～１８までの数字が出力されていて、下にスクロールしていくほどに小さいものは下へ下へと並び替えが行われていることがわかりました。

9行目で要素数を20と指定してあるのに、１９までの数字しか出ないのはなぜでしょうか？

**Comment by nsc-kiwamu** [comment]
ソースコードにコンソール出力を追加したものをpushしたので
確認してみてください。
